client/src/features/Event/Detail/hooks/useBouncingMessages.ts

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+
+type BouncingMessage = {
+  id: number;
+  message: string;
+  x: number;
+  y: number;
+  moveX: number;
+  moveY: number;
+};
+
+const POKE_MESSAGES = ['👆 콕!', '쿡ㅋ', '콕 찌르기', '콕콕콕', '👆👆👆'] as const;
+const DISTANCE = 80;
+
+export const useBouncingMessages = () => {
+  const [bouncingMessages, setBouncingMessages] = useState<BouncingMessage[]>([]);
+
+  const addBouncingMessage = (event: React.MouseEvent) => {
+    const rect = (event.target as HTMLElement).getBoundingClientRect();
+    const randomStartX = rect.left + Math.random() * rect.width;
+
+    const id = Date.now();
```

Comment:
p1. 아무래도 밀리세컨즈라 id가 겹칠일은 잘 없겠죠??
호오옥시 모른다면 

```suggestion
   const id = Date.now() + Math.random();
```
이런것도 괜찮을 수 있을것같아요